README.md

```diff
@@ -1,28 +1,157 @@
-# Jaeger OpenShift Templates
+# Getting started with the OpenShift template
 
-Support for deploying Jaeger into OpenShift.
+## Development setup
 
-## Install OpenShift
-If you are new to OpenShift these links help you to get it running:
-* https://install.openshift.com/
-* https://github.com/minishift/minishift
-
-## All-in-one template
-Template with in-memory storage with a limited functionality for local testing and development. 
+This template uses an in-memory storage with a limited functionality for local testing and development.
 Do not use this template in production environments.
 
 To directly install everything you need:
 ```bash
-oc process -f https://raw.githubusercontent.com/jaegertracing/jaeger-openshift/<version>/all-in-one/jaeger-all-in-one-template.yml | oc create -f -
-oc delete all,template -l jaeger-infra    # to remove everything
+oc process -f https://raw.githubusercontent.com/jaegertracing/jaeger-openshift/master/all-in-one/jaeger-all-in-one-template.yml | oc create -f -
 ```
 
-## Troubleshooting
-Tracer sometimes fails to resolve Jaeger's address. In such a case run:
+To cleanup and remove everything, just run:
 ```bash
-sudo iptables -F
+oc delete all,template,daemonset -l jaeger-infra
+```
+
+## Production setup
```

Comment:
@jstickler , is this something you could help us with? The challenge here is to have a minimum set of instructions where we would allow users to quickly try that out, but also providing enough information for an admin to properly install it in production.